build_tools/azure/py38_conda_defaults_openblas_environment.yml

```diff
@@ -0,0 +1,20 @@
+channels:
+  - defaults
+dependencies:
+  - python=3.8
+  - numpy=1.17.3  # min
+  - blas[build=openblas]
+  - scipy=1.3.2  # min
+  - cython
+  - joblib
+  - threadpoolctl=2.2.0
+  - matplotlib=3.1.2  # min
+  - pandas
```

Comment:
I find it weird to have numpy / scipy / matplotlib configured to `min` but not pandas.

I think we should have consistently `min`-labeled runtime dependencies in such builds.